janus_core/calculations/md.py

```diff
@@ -372,6 +391,8 @@ def __init__(
         self.post_process_kwargs = post_process_kwargs
         self.correlation_kwargs = correlation_kwargs
         self.seed = seed
+        self.enable_progress_bar = enable_progress_bar
+        self.progress_bar_update_every = progress_bar_update_every
```

Comment:
We may want a slightly more concise name if possible. `update_progress_every` maybe?